plot_test.go

```diff
@@ -57,10 +57,10 @@ func TestLegendAlignment(t *testing.T) {
 		},
 		&recorder.Fill{
 			Path: vg.Path{
-				{Type: vg.MoveComp, X: 80, Y: 30},
-				{Type: vg.LineComp, X: 80, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 30},
+				{Type: vg.MoveComp, Pos: vg.Point{X: 80, Y: 30}},
```

Comment:
OK. You've convinced me to leave it as `vg.Point{X: 80, Y: 30}`. It's not so bad, and it's one less function we have to expose.